views/layouts/main.handlebars

```diff
@@ -4,10 +4,10 @@
 	<meta charset="UTF-8">
 		<title>Games</title>
 		<script src="https://code.jquery.com/jquery.js"></script>
-        <link rel="stylesheet" href="../assets/stylesheets/bootstrap.min.css"></script>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 		<script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
 		<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
-		<link rel="stylesheet" href="../assets/stylesheets/style.css" type="text/css" />
```

Comment:
VERY VERY IMPORTANT!!! I believe this won't royally screw up everything as far as server-side code goes, but because this is on the main.handlebars which goes out to all files please make sure I didn't mess something up by doing these changes. -Sam